cicd/pipelines/creating-applications-with-cicd-pipelines.adoc

```diff
@@ -63,13 +63,20 @@ include::modules/op-creating-webhooks.adoc[leveloffset=+1]
 
 include::modules/op-triggering-a-pipelinerun.adoc[leveloffset=+1]
 
+include::modules/op-enabling-monitoring-of-event-listeners-for-triggers-for-user-defined-projects.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[Enabling monitoring for user-defined projects]
+
```

Comment:
On line 2 of this file, even though it is outside the scope of this PR, please replace `[id='creating-applications-with-cicd-pipelines']` with `[id="creating-applications-with-cicd-pipelines"]`